packages/vite/src/node/ssr/runtime/__tests__/server-source-maps.spec.ts

```diff
@@ -52,25 +52,25 @@ describe('vite-runtime initialization', async () => {
       resolvePath(import.meta.url, './fixtures/throws-error-method.ts'),
       (code) => '\n\n\n\n\n' + code + '\n',
     )
-    runtime.moduleCache.clear()
-    server.moduleGraph.invalidateAll()
+    runner.moduleCache.clear()
+    server.environments.ssr.moduleGraph.invalidateAll() // TODO: environment?
```

Comment:
```suggestion
    server.environments.ssr.moduleGraph.invalidateAll()
```
Probably this TODO is stale?